core/test-runtime/client/Cargo.toml

```diff
@@ -9,6 +9,7 @@ generic-test-client = { package = "substrate-test-client", path = "../../test-cl
 primitives = { package = "substrate-primitives", path = "../../primitives" }
 runtime = { package = "substrate-test-runtime", path = "../../test-runtime", default-features = false }
 runtime_primitives = { package = "sr-primitives", path = "../../sr-primitives" }
+parity-codec = { version = "4.1.1", default-features = false, features = ["derive"] }
```

Comment:
I'm surprise the `default-features = false` here doesn't require adding "parity-codec/std" in features::std (after "runtime/std" below)

Reply:
that is an interesting omission from my part.